README.md

```diff
@@ -22,90 +22,57 @@
 
 ## About
 
-### `payjoin`
+`payjoin/rust-payjoin` contains multiple crates intended to assist the implementation of Payjoin functionality as defined in [BIP 78: Payjoin V1 (Synchronous)](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki) and [BIP 77: Payjoin V2 (Asynchronous)](https://github.com/bitcoin/bips/blob/master/bip-0077.md), and the deployment of OHTTP relays and Payjoin directories which facilitate Async Payjoin transactions.
 
-The Payjoin Dev Kit `payjoin` library implements both [BIP 78 Payjoin V1](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki) and [BIP 77 Payjoin V2](https://github.com/bitcoin/bips/blob/master/bip-0077.md).
+Find the description of each crate below.
 
-### `payjoin-cli`
+### [`payjoin`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin)
 
-The [`payjoin-cli`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-cli) crate performs no-frills Payjoin as a reference implementation using Bitcoin Core wallet.
+The main Payjoin Dev Kit library which provides tools for implementing both Payjoin V2 (Asynchronous) and V1 (Synchronous). For Payjoin V2, the library also contains Payjoin session persistence support and IO utilities for interacting with OHTTP relays.
 
-### `payjoin-directory`
+**Disclaimer: This crate has not been reviewed by independent Rust and Bitcoin security professionals (yet). Use at your own risk.**
 
-The [`payjoin-directory`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-directory) crate implements the Payjoin Directory store-and-forward server required for Payjoin V2's asynchronous operation.
+### [`payjoin-cli`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-cli)
 
-### `payjoin-test-utils`
+A CLI tool which performs no-frills Payjoin. It is a reference implementation of the Payjoin Dev Kit which uses a Bitcoin Core wallet.
 
-The [`payjoin-test-utils`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-test-utils) crate provides commonly used testing fixtures such as a local OHTTP relay and payjoin directory, bitcoind node and wallets, and official test vectors.
+### [`ohttp-relay`](https://github.com/payjoin/rust-payjoin/tree/master/ohttp-relay)
 
-### `payjoin-ffi`
+A Rust implementation of an Oblivious HTTP (OHTTP) relay resource.
```

Comment:
It's worth mentioning high-level bip77 specific changes here and definitely in the ohttp-relay readme. TODO